README.md

```diff
@@ -4,9 +4,14 @@ Repository for the Game Mechaenetia
 This is my first real public Project, so I need to learn quite a lot about managing things.
 Other than that, just having fun learning and creating things, and hope everyone else will too. ^^
 
+If you want to know what is going on, or are interested in the Community, check out the Forum:
+https://forum.mechaenetia.com
+
 
 TODO:
 
+Stop getting distracted by Factorio and GT6.
```

Comment:
^.^;

Reply:
Yeah, I'm having a Factorio problem. I realized there is so many things I didn't try out in vanilla. XD